### PR TITLE
[AD-310] Ariadne Vty wallet

### PR DESCRIPTION
**Pull request by goodsoft**
`stack exec disciplina-wallet-vty -- --wallet-server 127.0.0.1:4020`

- [x] Copy Vty library from Ariadne
- [x] Make it compile
- [x] Show list of accounts
- [x] Selecting account from the list
- [x] Add new account form
- [x] Balance info
- [x] Send transaction form

### CODE REVIEW

File: wallet/app-vty/Glue.hs [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+          )
+      UiRestoreWallet UiRestoreWalletArgs{..} -> do
+        Right $ Knit.ExprProcCall
+          (Knit.ProcCall "restore-key" $
+            [ Knit.ArgKw "secretkey" . Knit.ExprLit . Knit.toLit . Knit.LitString $ urwaMnemonic
+            ] ++
+            optString "name" urwaName ++
+            optString "pass" urwaPassphrase
+          )
+      _ -> Left "Not implemented yet"
```
Comment (volhovm) [inline wallet/app-vty/Glue.hs]: By the way, which commands are missing now?

Comment (goodsoft) [reply wallet/app-vty/Glue.hs]: New account, new address (for HD-wallets), rename, remove...

Comment (volhovm) [reply wallet/app-vty/Glue.hs]: Ah ok, these constructors are imported (copypasted) from ariadne i guess?

Comment (goodsoft) [reply wallet/app-vty/Glue.hs]: Yes.

**Pull request merged**